smartmon.py

```diff
@@ -85,7 +85,7 @@ def type(self):
 
     @property
     def base_labels(self):
-        return {'disk': self.path}
+        return {'device': self.path, 'disk': self.type.partition("+")[2] or "0"}
```

Comment:
nit: standardise on single quotes for keywords, machine-readable strings etc, and only use double quotes for natural language strings (since they may contain apostrophes).